dms_version/models/version.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2017-2020 MuK IT GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import base64
+import logging
+import zlib
+from collections import defaultdict
+
+from odoo import _, api, fields, models, tools
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import bsdiff4
+except ImportError:
+    bsdiff4 = False
```

Comment:
```suggestion
import bsdiff4
```

Similar to previous change.